indico/legacy/pdfinterface/latex_templates/inc/document.tex

```diff
@@ -14,6 +14,10 @@
 
 \usepackage[english]{babel}
 \usepackage[final, babel]{microtype}
+\usepackage{xeCJK}
+\setCJKmainfont{Noto Serif CJK JP}
+\setCJKsansfont{Noto Sans CJK JP}
+\setCJKmonofont{Noto Sans Mono CJK JP}
```

Comment:
Just one thing - could you change the font paths to use the indico fonts rather than the system fonts?

```suggestion
\setCJKmainfont{NotoSerifCJKjp-VF.ttf}[Path = \VAR{font_dir | rawlatex}]
\setCJKsansfont{NotoSansCJKjp-VF.ttf}[Path = \VAR{font_dir | rawlatex}]
\setCJKmonofont{NotoSansMonoCJKjp-VF.ttf}[Path = \VAR{font_dir | rawlatex}]
```